parity/params.rs

```diff
@@ -184,6 +185,7 @@ impl Default for AccountsConfig {
 			testnet: false,
 			password_files: Vec::new(),
 			unlocked_accounts: Vec::new(),
+			enable_hardware_wallets: false,
```

Comment:
It is not needed for most use cases of account management such as creating or listing accounts.